src/UIComponents/SaveModel.jsx

```diff
@@ -83,6 +83,7 @@ class SaveModel extends Component {
                         onChange={event =>
                           this.handleChange(event, field.id, field.isColumn)
                         }
+                        maxLength={field.id === "name" ? ModelNameMaxLength : ""}
```

Comment:
Shoudl we also enforce length limits on the other fields?

Reply:
The other fields are stored as metadata, which is a text column in the database and has a much higher limit of 65,535 characters, so I think it's ok not to set a max length for the other fields.

Reply:
I feel like we've occasionally seen people paste an entire work of Shakespeare as a field in a form (for real) so it might be convenient for us to limit these to sane sizes, but we can ponder for a future work item.